src/MqttMailingService.cpp

```diff
@@ -1,27 +1,33 @@
 #include "MqttMailingService.h"
 #include <WiFi.h>
 
+#ifndef WIFI_CHECK_INTERVAL_MS
+#define WIFI_CHECK_INTERVAL_MS 10000
+#endif
+
+
 const char* TAG = "MQTT Mail";
 esp_mqtt_client_handle_t MqttMailingService::mEspMqttClient = nullptr;
 
 const uint8_t ssl_cert[] =
     "------BEGIN CERTIFICATE-----\n" MQTT_BROKER_CERTIFICATE_OVERRIDE
     "\n-----END CERTIFICATE-----";
 
+const uint8_t MAX_URI_LENGTH = 63;
+const uint8_t DEFAULT_MQTT_EVENT_LOOP_SIZE = 20;
+
+
 MqttMailingService::MqttMailingService() {
     mState = MqttMailingServiceState::UNINITIALIZED;
     // Apply default parameters
-    strncpy(mBrokerFullURI, MQTT_BROKER_FULL_URI_OVERRIDE, 64);
-    strncpy(mLwtTopic, MQTT_LWT_TOPIC_OVERRIDE, 127);
-    strncpy(mLwtMessage, MQTT_LWT_MSG_OVERRIDE, 255);
-    mSslCert = (char*)ssl_cert;
-#pragma clang diagnostic push
-#pragma ide diagnostic ignored "Simplify"
-#pragma ide diagnostic ignored "UnreachableCode"
-    mUseSsl = MQTT_USE_SSL_OVERRIDE == 0 ? false : true;
-#pragma clang diagnostic pop
-    mQos = MQTT_QOS_OVERRIDE;
-    mRetainFlag = MQTT_RETAIN_FLAG_OVERRIDE;
+    strncpy(mBrokerFullURI, "mqtt://mymqttbroker.com:1883", MAX_URI_LENGTH);
```

Comment:
I think it doesn't make sense to initialize mqtt broker URI with garbage, since this can't work.
I would rather leave the broker URI uninitialized as empty string.